custom-namespaces.yaml

```diff
@@ -1392,6 +1392,14 @@ monitoring:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.monitoring.looker_usage_models
+    shredder_run_stats:
```

Comment:
Can you move these up to where the existing shredder views are defined so it's easier to see what exists?  Also can you add `moz-fx-data-shared-prod.monitoring.shredder_per_table_stats` while you're here?